visual_testing_wdio/wdio.chrome.conf.js

```diff
@@ -0,0 +1,35 @@
+const baseConfig = require('./wdio.conf.js').config
+const headlessModeArgs = [
```

Comment:
Remove headless configs since you are not using it. Keep the code as cleaner as possible without any additional details that distract viewer from the main purpose of this codebase